plugins/ui/test/deephaven/ui/test_ui_table.py

```
@@ -131,41 +131,6 @@ def test_can_search(self):
             },
         )
 
-    def test_quick_filter(self):
```

Can we add some tests back in?
One thing we should be testing is that if the column doesn't exist on the table, then we should throw... in your example in the PR description, you have:
```python
from deephaven import empty_table
import deephaven.ui as ui

t = empty_table(1000).update("X=ii")

ui_table = ui.table(t, quick_filters={"X": ">50", "Y": "=20"}, show_quick_filters=True)
ui_with_search = ui.table(t, show_search=True)
ui_with_search_and_filter = ui.table(t, show_search=True, show_quick_filters=True)
```
...But there is no column `Y`, even though you specify that filter on `ui_table`. I think we should raise an error right there.

---

Sure

---

But still want tests